docs/docs/20-quickstart.md

```diff
@@ -568,269 +546,42 @@ the previous section.
     version of the `public.ecr.aws/nginx/nginx` container image.
     :::
 
-1. We'll use it later, so save the ID of the `Freight` to an environment
-   variable:
-
-    ```shell
-    export FREIGHT_ALIAS=$(kargo get freight --project kargo-demo --output jsonpath={.alias})
-    ```
-
-1. Now, let's _promote_ the `Freight` into the `test` `Stage`:
-
-    ```shell
-    kargo promote --project kargo-demo --freight-alias $FREIGHT_ALIAS --stage test
-    ```
-
-    Sample output:
-
-    ```shell
-    promotion.kargo.akuity.io/test.01j2wbtrym4r3tktv38qzteh5h.7a6e91f promotion created
-    ```
-
-    Query for `Promotion` resources within our project to see one has been
-    created:
+### Promote to Test
 
-    ```shell
-    kargo get promotions --project kargo-demo
-    ```
+1. To promote `Freight` to the `test` `Stage`, select the existing `Freight` from
+    the Freight Timeline above. Click the three dots, choose <Hlt>Promote</Hlt>,
+    and then select <Hlt>Promote</Hlt> that appeared on the `test` `Stage`.
+ 
+    The promotion may briefly appear in a `Pending` state, but it will likely
+    transition to `Running`, or even `Succeeded` within moments, as shown below:
 
-    Our `Promotion` may briefly appear to be in a `Pending` phase, but more than
-    likely, it will almost immediately be `Running`, or even `Succeeded`:
+    ![Kargo-dashboard-screenshot](../static/img/kargo-dashboard-promotion.png)
 
-    ```shell
-    NAME                                      SHARD   STAGE   FREIGHT                                    PHASE       AGE
-    test.01j92m3mx7jn49rc5p62tmnsf1.b2ceb4f           test    b2ceb4f821be8b682861c579171874028e4275fd   Succeeded   57s
-    ```
+    You'll notice a **_check mark_**, indicating a <Hlt>successful promotion</Hlt>.
```

Comment:
There may be several spots to fix this throughough, but please use `<Hlt>` only in _direct reference_ to some UI element. (There is no text anywhere on the screen that says "successful promotion".)